config/src/main/java/tech/pegasys/pantheon/config/JsonGenesisConfigOptions.java

```diff
@@ -21,7 +21,7 @@
 
   private static final String ETHASH_CONFIG_KEY = "ethash";
   private static final String IBFT_CONFIG_KEY = "ibft";
-  private static final String REVISED_IBFT_CONFIG_KEY = "revisedibft";
+  private static final String REVISED_IBFT_CONFIG_KEY = "ibft2";
```

Comment:
probably should change the constant name (replace "Revised" with something else)

Reply:
yep the constant should renamed as well

Reply:
done